memdown.js

```diff
@@ -90,6 +90,7 @@ MemIterator.prototype._next = function (callback) {
   var value
 
   if (this._done++ >= this._limit) return setImmediate(callback)
+  if (!this._tree) return setImmediate(callback)
```

Comment:
Don't need this line, because `this._tree.valid` (on the next line) will be false thanks to:

https://github.com/Level/memdown/blob/54dd8e7786776ec45c8917edaa329ce8b665c3fb/memdown.js#L146-L147

Reply:
cool - fixed. Sorry I had some issues with vscode complaining it couldn't save after I pulled.